test/system/103-run.bats

```diff
@@ -19,6 +19,29 @@ teardown() {
   cd "$CURDIR" || return 1
 }
 
+@test "run: containerstart: Check container starts without issues" {
```

Comment:
I am not a fan of the `context: context: testname` syntax. Maybe we could be a bit radical and shuffle the tests a bit to give more flexibility for tests like this.

Reply:
I can remove the "run:" and just leave the containerstart? The other option is to create a new file for container tests and put it before the run ones

Reply:
Maybe `start` would be enough. And I'd put it into its own file.

Reply:
Done!

Reply:
After looking at https://github.com/containers/toolbox/pull/839 I think using `container` instead of `start` will be better. That way the two tests could live in the same file since they are both related to correct behaviour and state of toolbox containers.

Reply:
Makes sense :)